.github/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directories:
```

Comment:
When I tried it didn't work in the apm-pipeline-library. Maybe it was something wrong in my end. Let's see if it works here 🙏